src/views/Home.js

```diff
@@ -18,7 +22,7 @@ class Home extends React.Component {
   static fetchData(dispatch) {
     return Promise.all([
       dispatch(fetchHearingList("nextClosingHearing", "/v1/hearing/", {next_closing: (new Date().toISOString())})),
```

Comment:
I would get rid of the next closing Hearing section altogether, as this will be shown in the grid anyway.

However, the next closing Hearing(s) should be highlighted within the grid, one way or another. Perhaps a red text "n hours left" for any Hearing closing within the next week or two days or so, like suggested in the wireframes.